source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModel/AdaptiveCardsSharedModel.vcxproj

```diff
@@ -22,7 +22,7 @@
     <VCProjectVersion>15.0</VCProjectVersion>
     <ProjectGuid>{430407C5-059C-48E8-9260-62A12D30F81C}</ProjectGuid>
     <RootNamespace>AdaptiveCardsSharedModel</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.14393.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
```

Comment:
Shouldn't this be the target version and min version be 15063?

Reply:
Hmm... I just updated via VS, which didn't offer me a min version option.

Reply:
Oh, yeah, that's not a UWP, so the WindowsTargetVersion should probably be consistent with the other target versions. Which means which SDK to use.